CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing
+## Advice for new contributors
+Small tasks get merged easier as the scope of changes won't require wider review. We recommend starting with these.
+
+Always create an `Issue` before doing a `Pull Request`. This helps us track our progress easier. We don't accept `Pull Requests` on their own.
+
+Search through already labeled Issues such as `area:contracts` and `kind:feature`. You will get those merged easier as they are examined and identified already by one of our core committers.
+
+If you want to start solving the `Issue`, please comment in it with something like: "I would like to take a stab at this". This will help us coordinate and distribute the workload.
+
+See below for more guidelines on [Pull Requests](#pull-requests).
+## Developer setup
```

Comment:
You already have a lot of this in the readme, why repeat?

Reply:
Oh I just realized you moved it.. Why?  Readme should contain quickstart stuff

Reply:
Sorry didn't understand, a lot of what?

Reply:
Developer setup / quickstart stuff.. should be right in the front of the readme so people can get started (thats what I usually see in OSS repos, it's only when they get to be big repos that they start separating out the different sections)

Reply:
Well the reason for me was that usually when you have some sort of `quickstart` you always have an example how to use the library. It's a bit unclear how you can really use this library still so I didn't include it to the `README.md`. It made sense however for `CONTRIBUTING.md` because someone could be interested in contributions even though they don't use the library.

Reply:
You could write something like:
## Quickstart
Use Docker, run `bin/prelaunch` and `bin/test` but not really sure if that's useful. For contributing however the `bin/test` shows that you have setup it correctly and you can start contributing.

Reply:
That's fair .. maybe in readme we can have a "heres a quick way to deploy it' ?  Just to get em started.  It's fairly bare now

Reply:
Yeah, I don't know what's the use of someone deploying our token?

Reply:
Yeah agree the test should be in contributing .. how bout just add `bin/run` to a brief quickstart?

Reply:
In a way `bin/run` migrates it into local network.

Reply:
Yea that makes sense, also could add the non docker version

Reply:
Yeah I think the run is handy, you get a working deployed token in a ganache as a quickstart. thats handy